src/lpython/semantics/python_attribute_eval.h

```diff
@@ -331,7 +331,9 @@ struct AttributeHandler {
             throw SemanticError("'get' takes atleast 1 and atmost 2 arguments",
                     loc);
         }
-        ASR::ttype_t *type = ASRUtils::expr_type(s);
+        ASR::ttype_t *type = ASRUtils::type_to_str(ASRUtils::expr_type(s)) == "dict const"
+                                 ? ASRUtils::get_contained_type(ASRUtils::expr_type(s))
+                                 : ASRUtils::expr_type(s);
```

Comment:
This does not seem to be the correct approach. See https://github.com/lcompilers/lpython/pull/2570#discussion_r1515196094.

If you want to check for `const` in type, use `ASR::is_a<ASR::Const_t>()`. 
If you want to get past the const and get to the actual type, use `ASRUtils::type_get_past_const()`.

Reply:
@Shaikh-Ubaid I had started with the above approach, but then switched as it threw this error every time:
![image](https://github.com/lcompilers/lpython/assets/151380951/15c4c1b3-f3fc-4a52-a8d1-29561512eab2)

Reply:
The problem is resolved. My bad, I had missed the pointer asterisk. I am pushing the changes stated by you. :+1: Thank you very much!